code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -2666,96 +2666,979 @@ Follow by example and make good judgement based on length which list to include
+
+	scratch_chest_left
+		name = "Scratch, Chest (Left)"
+		icon_state = "scratch_chest_l"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	scratch_chest_right
+		name = "Scratch, Chest (Right)"
+		icon_state = "scratch_chest_r"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	skull_paint
+		name = "Skull Paint"
+		icon_state = "skull"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_belly
+		name = "Tattoo (Belly)"
+		icon_state = "tat_belly"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_campbell_leftarm
+		name = "Tattoo (Campbell, Left Arm)"
+		icon_state = "tat_campbell"
+		body_parts = list("l_arm")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_campbell_rightarm
+		name = "Tattoo (Campbell, Right Arm)"
+		icon_state = "tat_campbell"
+		body_parts= list("r_arm")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_campbell_leftleg
+		name = "Tattoo (Campbell, Left Leg)"
+		icon_state = "tat_campbell"
+		body_parts= list("l_leg")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_campbell_rightleg
+		name = "Tattoo (Campbell, Right Leg)"
+		icon_state = "tat_campbell"
+		body_parts= list("r_leg")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_forrest_left
+		name = "Tattoo (Forrest, Left Eye)"
+		icon_state = "tat_forrest_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_forrest_right
+		name = "Tattoo (Forrest, Right Eye)"
+		icon_state = "tat_forrest_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_hive
+		name = "Tattoo (Hive, Back)"
+		icon_state = "tat_hive"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_heart
+		name = "Tattoo (Heart, Chest)"
+		icon_state = "tat_heart"
+		body_parts = list("chest")
+		species_allowed = list("Human","Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_hunter_left
+		name = "Tattoo (Hunter, Left Eye)"
+		icon_state = "tat_hunter_l"
+		body_parts = list("head")
+		species_allowed = list("Human","Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_hunter_right
+		name = "Tattoo (Hunter, Right Eye)"
+		icon_state = "tat_hunter_r"
+		body_parts = list("head")
+		species_allowed = list("Human","Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_jaeger_left
```

Comment:
Metal Gear Solid Reference

Reply:
With the new changes that break up the MGS pattern (Campbell and Silverburgh notwithstanding), would you still say that a rename is necessary?

Reply:
I guess this is acceptable now